flowey/flowey_lib_common/src/run_cargo_nextest_run.rs

```diff
@@ -578,7 +578,7 @@ impl FlowNode for Node {
                             format!(
                                 r#"
                                 - publish: $({path_var})
-                                  artifact: crash-dumps-{friendly_name}-$(Build.BuildNumber)-{i}"#,
+                                  artifact: {friendly_name}-crash-dumps-{i}"#,
```

Comment:
two things

1. having a static-prefix here seems better than a loose "friendly_name" prefix
2. is it actually safe to remove the `Build.BuildNumber` from this name? IIRC, that was there to avoid name collisions when the same pipeline job is rerun

Reply:
1. All the other artifacts start for {arch}-{os}-.... so I changed this for consistency.
2. It wasn't clear to me from the docs that `$(Build.BuildNumber)` is what we want. I am trying to figure out what we do want and make it consistent- it looks like we want `github.run_attempt` on github.

Reply:
1. OK
2. @smalis-msft might recall why $(Build.BuildNumber) is being used here

Reply:
I think it was to support retries?